cmd/podman/machine/machine.go

```diff
@@ -68,6 +68,40 @@ func autocompleteMachineSSH(cmd *cobra.Command, args []string, toComplete string
 	return nil, cobra.ShellCompDirectiveDefault
 }
 
+// autocompleteMachineCp - Autocomplete machine cp command.
+func autocompleteMachineCp(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	if len(args) < 2 {
+		if i := strings.IndexByte(toComplete, ':'); i > -1 {
+			// the user already set the machine name, so don't use the host file autocompletion
```

Comment:
Can you add a comment here that it would be nice to offer VM file path file completion. I am not expecting you to add that right now but I know scp does complete remote paths so it would be cool if someone takes the time to do that in the future.